### ISSUE TITLE
Filter for videos with subtitles

### ISSUE THREAD

**Issue by blubbsy** [mediathekview/mediathekviewweb#74]
Hi,

i'm a regular suer of the different mediatheks to improve my german skills and i would appreciate if it would be possible to filter videos which own subtitles. at the moment it is always a pain in the ass to search in the official mediathek for videos with subtitles, as it is necessary to open the video first to see the option.

Would it be possible to do this? or is it already possible?

sincerely,
blubbsy

**Comment by bagbag** [comment]
You cannot filter now, but you can see it on the download overlay as "UT" (for UnterTitel)

<img width="406" alt="unbenannt" src="https://cloud.githubusercontent.com/assets/4116942/25566916/6dc68a74-2de3-11e7-9343-11a4fba628dc.PNG">

I could add filtering for that in MVW 2.0, but for current version, this has to be enough for you.

**Comment by blubbsy** [comment]
Yeah i know about this (which is already great). For MVW 2.0 it would be awesome!

**Comment by stefan123t** [comment]
Yes, as well on the contrary it might also be wanted to exclude search results with subtitles:

E.g. arte has three different versions per language / channel (!arte.de, !arte.fr, etc.)

https://mediathekviewweb.de/#query=!arte.de%20Homo%20sapiens

* Homo sapiens - Vom wahren Ursprung des Menschen
* Homo sapiens - Vom wahren Ursprung des Menschen (Audiodeskription)
* Homo sapiens - Vom wahren Ursprung des Menschen (mit Untertitel)

Here the `(Audiodeskription)`, `(mit Untertitel)` yields a total of three results per site, while only one may be wanted.

Fun fact, the french `!arte.fr` has the same extensions, i.e. `(Audiodeskription)` / `(mit Untertitel)` in german despite being for french titles.
Is this due to the scraper running in de_de locale ?
